### ISSUE TITLE
clarify Interval formatting doc (and/or improve Interval formatting)

### ISSUE THREAD

**Issue by wkeese** [moment/luxon#691]
The doc on https://moment.github.io/luxon/docs/class/src/interval.js~Interval.html states that:

> To convert the Interval into other representations, see toString, toISO, toISODate, toISOTime, toFormat, and toDuration.

However, the link to `toFormat()` goes to the DateTime documentation page (it took me a while to realize this), and that page says:

> You may not want this. See toLocaleString for a more flexible formatting tool. 

But Interval.toLocaleString() actually prints out ISO format.

I realize as written in #325 that range formatting isn't supported properly, so this ticket can be just about updating the doc... although I do wonder why you don't use Intl.formatRange() and then when unsupported, fallback to `interval.start.toLocalString(options) + " - " + interval.end.toLocalString(options)`.

**Comment by icambron** [comment]
Yeah, that's probably just copied and pasted from the DateTime one. I agree the doc should be fixed.

I completely agree about format range and the fallback; I actually hadn't realized that `formatRange` was implemented anywhere. It seems to work for me in at least Chrome though. Would take a PR for that.

**Comment by GillesDebunne** [comment]
The doc fixes are already on the 2.0 branch, commits `6273a494` and `4ece63b0`.

`formatRange` is only available on Chrome according to [CanIUse](https://caniuse.com/#search=formatRange), which accounts for 64% of users.
